apps/benchmark/build.js

```diff
@@ -0,0 +1,40 @@
+const esbuild = require('esbuild');
+const ImportGlobPlugin = require('esbuild-plugin-import-glob').default;
+const AliasPlugin = require('esbuild-plugin-alias');
+const fs = require('fs');
+const path = require('path');
+const yargs = require('yargs');
+
+const shouldWatch = yargs.argv.watch;
+
+const outDir = path.resolve(__dirname, '../../dist/apps/benchmark/');
+
+esbuild
+  .build({
+    entryPoints: ['./src/index.tsx'],
+    outfile: path.join(outDir, 'bundle.js'),
+    minify: true,
+    bundle: true,
+    plugins: [
+      AliasPlugin({
+        '@griffel/core': path.resolve(__dirname, '../../packages/core/src/index.ts'),
+        '@griffel/react': path.resolve(__dirname, '../../packages/react/src/index.ts'),
+      }),
```

Comment:
```suggestion
```

We don't need this, https://esbuild.github.io/content-types/#tsconfig-json

Reply:
Done in [f7390ee](https://github.com/microsoft/griffel/pull/132/commits/f7390ee406a19eb46b63faf80116febb81ba84a6)